battlecode.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
I'm slightly worried the proliferation of shell scripts and batch files is getting confusing. Maybe we should come up with another naming scheme? Maybe `run_gui.sh`, `run_gui_nodocker.sh`, `battlecode.sh`?

Reply:
Ideally I think the battlecode.sh script should have a --web flag to avoid having several different shell scripts.

Reply:
Main problem is that the gui is aimed at people who don't use the terminal much and like to double-click things, which won't be possible if it requires arguments :/

Reply:
Ah. True. I suppose the bat script could just call battlecode.sh --web though, but maybe there isn't that much of a point then.